src/vs/workbench/contrib/search/common/searchModel.ts

```diff
@@ -1350,3 +1415,13 @@ function textSearchResultToMatches(rawMatch: ITextSearchMatch, fileMatch: FileMa
 		return [match];
 	}
 }
+
+export function arrayContainsElementOrParent(element: RenderableMatch, testArray: (RenderableMatch | SearchResult)[]): boolean {
```

Comment:
Does `| SearchResult` need to be in that type? If `SearchResult` is in that array, it's the parent of everything so this would always be true.

Reply:
I think this is due to another call to the function that was there before, but I could investigate why it needs this